en/dev_setup/dev_env_mac.md

```diff
@@ -29,6 +29,11 @@ To build other targets you will need to use a [different OS](../dev_setup/dev_en
 
 The installation of Homebrew is quick and easy: [installation instructions](https://brew.sh).
 
+Note that with PX4 make, the exact versions of tools like CMake are cached in the build. If you do a `brew update` it will automatically 
```

Comment:
Thanks @richtong 

80%+ of users will not need to know this when they first start using the build toolchain - they just install and everything works. They only need this after doing `brew update`, which will happen later, if at all.

My suggestion is to add a troubleshooting section at the end of the document and then have this note under its own heading.

What exactly do you see in the failure case of "your build can fail searching for executables in `/usr/local/Cellar` "?

Reply:
Ping